testing/run_tests.py

```diff
@@ -217,8 +217,8 @@ def EnsureDebugUnoptSkyPackagesAreBuilt():
     '--no-lto',
   ]
 
-  subprocess.check_output(gn_command, cwd=buildroot_dir)
-  subprocess.check_output(ninja_command, cwd=buildroot_dir)
+  print(subprocess.check_output(gn_command, cwd=buildroot_dir))
```

Comment:
nit: It may be helpful to bundle `print(subprocess.check_output` into a helper function. It's trivial on its own but since _all_ the process calls in the file are expected to follow this format I think it's relatively easy for one to slip through the cracks. Having a helper function wouldn't totally prevent that but it would make it less likely.